src/pages/AnnualReport/ClubAnnualReportForm/ClubAnnualReportForm.tsx

```diff
@@ -137,105 +193,146 @@ export const ClubAnnualReportForm = (props: Props) => {
                         </Col>
                     </Row>
                     <Row gutter={{ xs: 8, sm: 16, md: 24, lg: 32 }}>
-                        <Col className="gutter-row" span={12}>
-                            <Text strong={true}>Географія куреня. Осередки в Україні:</Text>
+                        <Col
+                            className="clubAnnualReportResponsiveCols"
+                            span={12}
+                        >
+                            <Text strong={true}>
+                                Географія куреня. Осередки в Україні:
+                            </Text>
                             <Form.Item
-                                className='w100'
-                                name='clubCenters'
-                                rules={validationSchema.textareaClubCenters} >
-                                <TextArea style={{  height: "100px" }} />
+                                className="w100"
+                                name="clubCenters"
+                                rules={validationSchema.textareaClubCenters}
+                            >
+                                <TextArea style={{ height: "100px" }} />
```

Comment:
it's highly recommended to set all the css properties to the other file and not to write it inside tsx files. So, please, remove all the css code from this file

Reply:
Ok, i will change it